src/cloud/social/monkey/process.js

```diff
@@ -0,0 +1,11 @@
+// Monkey patch for browserify's process shim, for ssh2-streams.
+
+const process = require('process');
+
+// Node.js 4.2 is an LTS release and, very roughly, is what the
+// browserify shims, e.g. Buffer, provide.
+process.version = '4.2.0';
```

Comment:
Just wondering if we also need to populate `.versions`, per https://github.com/defunctzombie/node-process/blob/4b996b1e3281aa38d46294f33b9772771f53272d/browser.js#L71